test/dist_autograd_test.py

```diff
@@ -418,8 +423,9 @@ def test_backward_autograd_engine_error(self):
                 # Run backwards, and validate we receive an error.
                 dist_autograd.backward([val.sum()])
 
-    @dist_init
-    @unittest.skip("Skipping this test temporarily since ProcessGroupAgent does not report errors on node failures")
+    @dist_init(sync_at_shutdown=False)
+    @unittest.skipIf(TEST_CONFIG.rpc_backend == RpcBackend.PROCESS_GROUP,
```

Comment:
What errors do you expect `ProcessGroupAgent` to report? Will timeout on send be sufficient?